api/npm.ts

```diff
@@ -80,7 +81,8 @@ async function pkgJson (pkg: string, tag = 'latest'): Promise<any> {
 }
 
 async function info (topic: string, pkg: string, tag = 'latest') {
-  const meta = await (tag === 'latest' && pkg[0] !== '@' ? npmMetadata(pkg) : pkgJson(pkg, tag))
+  // when a custom registry is defined, tag=latest or package is scoped use npmMetadata
+  const meta = await (process.env.NPM_URL || (tag === 'latest' && pkg[0] !== '@') ? npmMetadata(pkg) : pkgJson(pkg, tag))
```

Comment:
Will the private npm registry api support query tags other than `latest`?

Reply:
I'm testing using Artifactory and yes it does. Though it's a bummer NPM (at least public) does not. 

I wonder if instead of using unpkg one could just query the root package and pull the version out of the `versions` object?

I'll experiment with some things.

Reply:
> if instead of using unpkg one could just query the root package and pull the version out of the versions object?

I tried this method, it would be a huge json response for some long running packages like `webpack`